scaladoc-testcases/src/tests/snippetTestcase2.scala

```diff
@@ -9,7 +9,7 @@ trait Quotes2[A] {
       /**
         * SNIPPET(OUTERLINEOFFSET:13,OUTERCOLUMNOFFSET:10,INNERLINEOFFSET:6,INNERCOLUMNOFFSET:6)
         * ERROR(LINE:13,COLUMN:12)
-        * ```scala sc:compile
```

Comment:
Why do we need to change that script to `fail`?

Reply:
It's because the snippets that I've fixed, had wrong flags and were throwing errors on documentation generation. It caused the process to exit with failure and break the script execution